applications/mne_scan/plugins/babymeg/babymeg.cpp

```diff
@@ -122,7 +122,7 @@ BabyMEG::~BabyMEG()
 QSharedPointer<IPlugin> BabyMEG::clone() const
 {
     QSharedPointer<BabyMEG> pBabyMEGClone(new BabyMEG());
-    return pBabyMEGClone;
+    return std::move(pBabyMEGClone);
```

Comment:
I like this. Could you enforce this kind of behavior in all plugins' clone() methods defined in [plugins.pro](https://github.com/mne-tools/mne-cpp/blob/master/applications/mne_scan/plugins/plugins.pro)?

Reply:
Maybe. jk